InternTrack.Core.Api.Tests.Acceptance/Usings.cs

```diff
@@ -0,0 +1 @@
+using Xunit;
```

Comment:
Why's this file here?

Reply:
I am not sure why I have this file . Double checking training videos. Will delete this file if not needed